opencti-platform/opencti-graphql/src/database/session.js

```diff
@@ -112,14 +112,23 @@ export const killUserSessions = async (userId) => {
 };
 
 export const markSessionForRefresh = async (id) => {
+  const { store } = applicationSession;
   const currentSession = await getSession(id);
   if (currentSession) {
     const newSession = { ...currentSession, session_refresh: true };
-    await setSession(id, newSession, currentSession.expiration);
+    const sessId = id.includes(store.prefix) ? id.split(store.prefix)[1] : id;
+    store.set(sessId, newSession); // this will ensure the session is updated in the cache
+    // TODO check what to do with currentSession.expiration
+    // await setSession(id, newSession, currentSession.expiration);
```

Comment:
TODO ?

Reply:
And it can be merge with it or should it be investigated before merging ?

Reply:
@SouadHadjiat

Reply:
Yes I think we can merge it, I'll remove this TODO in my current PR on sessions